.github/CODEOWNERS

```
@@ -1,6 +1,6 @@
 # Team
 # @ryan-conway is the Team Lead and the others are team members
-* @AVI5HEK @chornerman @hoangnguyen92dn @Johnsonmaung @kaungkhantsoe @luongvo @manh-t @minhnimble @nyinyihtunlwin-dev @RkNaing @ryan-conway @sleepylee @thawzintoe-ptut @toby-thanathip @win-than-htike
+* @AVI5HEK @chornerman @eeeeaa @hoangnguyen92dn @Johnsonmaung @kaungkhantsoe @luongvo @manh-t @minhnimble @nyinyihtunlwin-dev @RkNaing @ryan-conway @sleepylee @thawzintoe-ptut @toby-thanathip @win-than-htike
```

Can we also remove Nyi Nyi too? 🙏🏻 

```suggestion
* @AVI5HEK @chornerman @eeeeaa @hoangnguyen92dn @Johnsonmaung @kaungkhantsoe @luongvo @manh-t @minhnimble @RkNaing @ryan-conway @sleepylee @thawzintoe-ptut @toby-thanathip @win-than-htike
```